cmd/utils/git.go

```diff
@@ -85,3 +88,20 @@ func GetRandomSHA(ctx context.Context, path string) string {
 	}
 	return string(b)
 }
+
+func IsOktetoRepo() (bool, error) {
```

Comment:
Can you add a test for this function including git URLs in their different forms (SSH and HTTPS)?